common/output_stream.cpp

```diff
@@ -110,7 +110,7 @@ std::string ToStringSpvSourceLanguage(SpvSourceLanguage lang) {
     case SpvSourceLanguageNZSL:
       return "NZSL";
 
-    case SpvSourceLanguageMax:
+    default:
       break;
   }
   // unhandled SpvSourceLanguage enum value
```

Comment:
instead of `return "???";` can we do something like `return "SpvSourceLanguage not found";`

Reply:
I return "Unknown" following @chaoticbob's suggestion in the other file.